src/jit/gentree.h

```diff
@@ -2107,7 +2107,7 @@ struct GenTreeLclVar: public GenTreeLclVarCommon
                             DEBUG_ARG(largeNode)),
             gtLclILoffs(ilOffs)
             {
-                assert(OperIsLocal(oper));
+                assert(OperIsLocal(oper) || OperIsLocalAddr(oper));
```

Comment:
This seems like a fundamental change: GT_LCL_var can now take a local addr? why?